config/serverless.security.yml

```diff
@@ -106,7 +106,7 @@ xpack.fleet.internal.registry.excludePackages: [
     'dga',
 
     # Unsupported in serverless
-    'cloud-defend',
+    'cloud_defend',
```

Comment:
It seems like this typo went completely unnoticed. Is there a way we can add some validation for this so that these kinds of things are detected during development?